examples/2d/mesh2d_manual.rs

```diff
@@ -1,3 +1,8 @@
+//! This example shows how to manually render 2d items using "mid level render apis" with  a custom
+//! pipeline for 2d meshes.
+//! It doesn't use the [`Material2d`] abstraction, but changes the vertex buffer to include vertex color
+//! Check out the "mesh2d" example for simpler / higher level 2d meshes
```

Comment:
```suggestion
//! It doesn't use the [`Material2d`] abstraction, but changes the vertex buffer to include vertex color.
//! Check out the "mesh2d" example for simpler / higher level 2d meshes.
```

Period at the end of sentence.